src/DynamoCoreWpf/UI/Converters.cs

```diff
@@ -3327,4 +3327,25 @@ public object ConvertBack(object value, Type targetType, object parameter, Syste
 
         #endregion
     }
+
+    /// <summary>
+    /// Converts a boolean to Visibility but inverted, then false will be Visible and true will be Collapsed
+    /// </summary>
+    [ValueConversion(typeof(bool), typeof(Visibility))]
+    public class InvertedBooleanToVisibilityConverter : IValueConverter
```

Comment:
I thought there is already a `BoolToVisibilityCollapsedConverter`? I dont think we need to create a new one here

Reply:
@QilongTang for this case I need a converted that that does the inverse operation, in the BoolToVisibilityCollapsedConverter if the value is **true** returns **Visible** and if is **false** returns **Collapsed**, for this case is the opposite, in the InvertedBooleanToVisibilityConverter  if is **true** then returns **Collapsed** (meaning that property IsDefault = true so the button should be collapsed) otherwise returns **Visible**.
Please let me know what you think about it.

Reply:
Then how about using `InverseBoolToVisibilityConverter`

Reply:
`InverseBoolToVisibilityCollapsedConverter ` sounds good

Reply:
@QilongTang I removed the converter that I added due that there was one already existing doing the same functionality.
https://github.com/DynamoDS/Dynamo/pull/12711/commits/73d3cadf7a8143c56ff8113f7fe75b1fe015061c